From 5050a1fd7cc015f9c88f5582dd8b52ecb7144468 Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Wed, 1 Oct 2014 17:40:35 -0400
Subject: [ethernet] cxgb4: Add device ID for new adapter and remove for dbg adapter

Message-id: <1412185238-11567-5-git-send-email-svemuri@redhat.com>
Patchwork-id: 96831
O-Subject: [RHEL7.1 PATCH BZ 1147683 4/7] cxgb4/cxgb4vf: Add device ID for new adapter and remove for dbg adapter
Bugzilla: 1147683
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: David S. Miller <davem@redhat.com>

Upstream : https://git.kernel.org/cgit/linux/kernel/git/davem/net-next.git/commit/drivers/net/ethernet/chelsio?id=56e03e51e61afc60d893be9647a8e3ad74155420

Signed-off-by: Hariprasad Shenai <hariprasad@chelsio.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 56e03e51e61afc60d893be9647a8e3ad74155420)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
index db807d6..0111175 100644
--- a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
@@ -282,6 +282,7 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4_pci_tbl) = {
  CH_DEVICE(0x5083, 4),
  CH_DEVICE(0x5084, 4),
  CH_DEVICE(0x5085, 4),
+ CH_DEVICE(0x5086, 4),
  CH_DEVICE(0x5401, 4),
  CH_DEVICE(0x5402, 4),
  CH_DEVICE(0x5403, 4),
@@ -309,6 +310,7 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4_pci_tbl) = {
  CH_DEVICE(0x5483, 4),
  CH_DEVICE(0x5484, 4),
  CH_DEVICE(0x5485, 4),
+ CH_DEVICE(0x5486, 4),
  { 0, }
 };
 
diff --git a/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c b/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
index a6a8598..b51365a 100644
--- a/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
@@ -2912,7 +2912,6 @@ static void cxgb4vf_pci_shutdown(struct pci_dev *pdev)
 
 static DEFINE_PCI_DEVICE_TABLE(cxgb4vf_pci_tbl) = {
  CH_DEVICE(0xb000, 0), /* PE10K FPGA */
- CH_DEVICE(0x4800, 0), /* T440-dbg */
  CH_DEVICE(0x4801, 0), /* T420-cr */
  CH_DEVICE(0x4802, 0), /* T422-cr */
  CH_DEVICE(0x4803, 0), /* T440-cr */
@@ -2934,7 +2933,6 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4vf_pci_tbl) = {
  CH_DEVICE(0x4880, 6),
  CH_DEVICE(0x4880, 7),
  CH_DEVICE(0x4880, 8),
- CH_DEVICE(0x5800, 0), /* T580-dbg */
  CH_DEVICE(0x5801, 0), /* T520-cr */
  CH_DEVICE(0x5802, 0), /* T522-cr */
  CH_DEVICE(0x5803, 0), /* T540-cr */
@@ -2962,6 +2960,7 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4vf_pci_tbl) = {
  CH_DEVICE(0x5883, 0),
  CH_DEVICE(0x5884, 0),
  CH_DEVICE(0x5885, 0),
+ CH_DEVICE(0x5886, 0),
  { 0, }
 };
 
-- 
1.7.1